app/helpers/admin/admin_cart_helper.rb

```diff
@@ -11,9 +11,37 @@ def set_class_to_active(status)
 
   def link_to_confirm_button(cart)
     if cart.status == "shipping"
-      link_to "確認收貨", confirm_admin_confirm_cart_path(cart), method: :post, class: "btn btn-success btn-lg"
+      link_to "確認收貨", confirm_admin_confirm_cart_path(cart), method: :post, class: "btn btn-success"
     elsif cart.status == "stock"
-      content_tag(:span, "確認已收貨", class: "label label-default")
+      content_tag(:span, "已收貨", class: "label label-default")
+    end
+  end
+
+  def admin_cart_item_quantity(cart_item)
+    if cart_item.admin_cart.status == "shipping"
+      render "update_quantity", item: cart_item
+    elsif cart_item.admin_cart.status == "stock"
+      content_tag :div do
+        content_tag(:p, "現貨數：#{cart_item.stock_item_quantity(cart_item.item_spec_id)}") +
```

Comment:
cart_item.stock_item_quantity(cart_item.item_spec_id)}
why should pass cart_item attr to cart_item ????